platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/CameraAnimatorActivity.java

```diff
@@ -237,6 +239,9 @@ protected void onStop() {
     for (int i = 0; i < animators.size(); i++) {
       animators.get(animators.keyAt(i)).cancel();
     }
+    if (set != null) {
+      set.cancel();
+    }
```

Comment:
if we cancel the set do we still need to keep the for loop that cancels them 1 by 1?

Reply:
Yes, animators spawned by the main animation under the FAB are not kept in the array.